devtools/conda-envs/conda_010X.yaml

```diff
@@ -0,0 +1,14 @@
+name: 010x-latest-deployment
+channels:
+  - conda-forge
+
+
+dependencies:
+    # Base depends
+  - openff-toolkit-examples =0.10
```

Comment:
I assume this will typically solve to `0.10.6` or newer and I don't think the complexity of more constraints is necessary, but we'd want to check a couple logs to make sure it doesn't somehow grab one of the buggy versions.

Reply:
Okay - looks like it doesn't now

Reply:
Yeah, I'd checked this manually during tests, and ensured that this action still finds the latest git tag via a separate command and ensures it matches with the installed version (just to be extra safe) :-)